packages/css-data/src/parse-css-value.ts

```diff
@@ -11,7 +11,7 @@ import {
 import { keywordValues } from "./__generated__/keyword-values";
 import { units } from "./__generated__/units";
 
-const cssTryParseValue = (input: string) => {
+export const cssTreeTryParseValue = (input: string) => {
```

Comment:
Why we need to expose `cssTree` (like internal implementation detail that we use csstree) whats wrong with previous name

Reply:
I just added `export` there ivan didn't change any name.

Reply:
But name has chaged 
cssTryParseValue 
cssTreeTryParseValue
<img width="503" alt="image" src="https://github.com/webstudio-is/webstudio/assets/5077042/ec1e3569-f050-49b6-b68c-957767045c09">